game/scripts/npc/heroes/techies.txt

```diff
@@ -5,9 +5,9 @@
   //=================================================================================================================
   "npc_dota_hero_techies"
   {
-    "Ability13"                                           "special_bonus_intelligence_35" // replaces special_bonus_mp_regen_3
```

Comment:
This talent didn't really add anything to the hero. INT talents are probably some of the worst talents any hero could have.

Reply:
It enables rightclick techies (it gives him dmg) and it gives him mana and mana regen. Having only mana regen as a talent is stupid.